UnityXrefMaps/XrefMap.cs

```diff
@@ -47,15 +56,30 @@ public void FixHrefs(string apiUrl, bool testUrls = false)
                     continue;
                 }
```

Comment:
We can remove this section if we remove `XrefMapReference.IsValid` if it's always true.